packages/web/src/components/gcds-nav-link/stories/gcds-nav-link.stories.tsx

```diff
@@ -0,0 +1,62 @@
+export default {
+  title: 'Components/Navigation link',
+
+  argTypes: {
+    // Props
+    href: {
+      name: 'href',
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' }
+      },
+      type: {
+        required: true
+      }
+    },
+    current: {
+      name: 'current',
+      control: 'boolean',
+      table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: false }
+      },
+    },
+
+    // Slots
+    default: {
+      control: {
+        type: 'text',
+      },
+      table: {
+        category: 'Slots | Fentes',
+      }
+    },
+  },
+};
+
+const Template = (args) => (`
+// Web Component (Angular, Vue)
```

Comment:
Can we update this comment to `<!-- Web component code (Angular, Vue) -->` so that it doesn't show in the preview?